src/backend/Coq.ml

```diff
@@ -44,55 +45,55 @@ module Make(S : Msat.S)(A : Arg with type hyp := S.clause
     Array.iter aux (Clause.atoms clause)
 
   let elim_duplicate fmt goal hyp _ =
-    (** Printing info comment in coq *)
-    Format.fprintf fmt
-      "(* Eliminating doublons. Goal : %s ; Hyp : %s *)@\n"
+    (* Printing info comment in coq *)
+    Format.fprintf fmt "(* Eliminating doublons. Goal : %s ; Hyp : %s *)@\n"
       (name goal) (name hyp);
-    (** Prove the goal: intro the atoms, then use them with the hyp *)
+    (* Prove the goal: intro the atoms, then use them with the hyp *)
     let m = clause_map goal in
     Format.fprintf fmt "pose proof @[<hov>(fun %a=>@ %s%a) as %s@].@\n"
-      (clause_iter m "%s@ ") goal (name hyp)
-      (clause_iter m "@ %s") hyp (name goal)
+      (clause_iter m "%s@ ") goal (name hyp) (clause_iter m "@ %s") hyp
+      (name goal)
```

Comment:
I understand this change from ocamlformat, but I still think the original version is more readable and understandable (one of the examples where regularity in code structure helps understand compared to compactness of code).